### PR TITLE
improve doc about fork memory usage

### PR DESCRIPTION
**Pull request by colinsurprenant**
Added a note about the plugin usage of fork and its memory usage consequence. 
/cc @karenzone

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -28,6 +28,12 @@ Notes:
 * The `command` field of this event will be the command run.
 * The `message` field of this event will be the entire stdout of the command.
 
+===== IMPORTANT
+
+The exec input ultimately uses fork to spawn a child process.
```
Comment (karenzone) [inline docs/index.asciidoc]: Use backticks for `fork`?  Otherwise LGTM.

Comment (colinsurprenant) [reply docs/index.asciidoc]:
```suggestion
The exec input ultimately uses `fork` to spawn a child process.
```

**Comment by colinsurprenant** [comment]
Added back ticks per @karenzone comment, bumped version.

**Review by karenzone — APPROVED** [review]
LGTM. Thanks for bumping the version so docs changes will be picked up.

**Comment by colinsurprenant** [comment]
v3.3.3 published. Thanks @karenzone !

**Pull request merged**